db/migrate/20151120153254_delete_bootable_interface.rb

```diff
@@ -0,0 +1,11 @@
+class DeleteBootableInterface < ActiveRecord::Migration
+  def up
+    Nic::Base.where(:type => 'Nic::Bootable').each do |nic|
```

Comment:
Indeed.  The PR currently defines a new model with the exact name needed, are you OK with that instead?  It makes me slightly uneasy, but the env should always be reloaded after a migration.